ThreadGroup/Views/TGSpinnerView.m

```diff
@@ -2,39 +2,41 @@
 //  TGSpinnerView.m
 //  ThreadGroup
 //
-//  Created by Patrick Butkiewicz on 6/10/15.
+//  Created by Anbita Siregar on 8/24/15.
 //  Copyright (c) 2015 Intrepid Pursuits. All rights reserved.
 //
 
 #import "TGSpinnerView.h"
 #import "UIView+Animations.h"
 
-@interface TGSpinnerView()
-
-@property (nonatomic, strong) UIView *nibView;
-@property (weak, nonatomic) IBOutlet UIImageView *spinnerLargeCounterClockwiseImageView;
-@property (weak, nonatomic) IBOutlet UIImageView *spinnerLargeClockwiseImageView;
-@end
-
 @implementation TGSpinnerView
 
-- (void)awakeFromNib {
-    [super awakeFromNib];
+- (instancetype)initWithFrame:(CGRect)frame clockwiseImage:(UIImage *)clockwiseImage counterClockwiseImage:(UIImage *)counterClockwiseImage {
```

Comment:
So in as/connectingUI, I actually completely change that whole method to `initWithContentMode:(TGNetworkPopupContentMode)contentMode` because I create content modes for each type of TGNetworkPopup. Should I still change it?